projections_test.go

```diff
@@ -402,6 +402,11 @@ func TestStrict(t *testing.T) {
 	if !errors.Is(result.Error, eventsourcing.ErrEventNotRegistered) {
 		t.Fatalf("expected ErrEventNotRegistered got %q", err.Error())
 	}
+
+	expected := "event not registered aggregate type: Person, reason: Born, global version: 1, event not registered"
+	if result.Error.Error() != expected {
+		t.Fatalf(`expected "%s", got %q`, expected, result.Error)
+	}
```

Comment:
Fragile assertion, the error type is checked above. One thing that could be verified is the `result.LastHandledEvent` but in this test it's never set.

```suggestion
```